src/main/scala/mesosphere/marathon/core/launchqueue/impl/AppTaskLauncherActor.scala

```diff
@@ -332,7 +332,11 @@ private class AppTaskLauncherActor(
       sender ! MatchedTasks(offer.getId, Seq.empty)
 
     case ActorOfferMatcher.MatchOffer(deadline, offer) =>
-      val newTaskOpt: Option[CreatedTask] = taskFactory.newTask(app, offer, tasksMap.values)
+      val newTaskOpt: Option[CreatedTask] = taskFactory.newTask(
+        app,
+        offer,
+        tasksMap.values.filterNot(_.getStatus.getState == TaskState.TASK_LOST))
```

Comment:
the task should only be in state if the reason/message indicate that it's unreachable, so I think a check for state `TASK_LOST` is sufficient